src/screens/Collections/Item.tsx

```diff
@@ -1,25 +1,30 @@
-import { Pressable, PressableProps, StyleSheet, Text } from 'react-native'
+import { Pressable, PressableProps, StyleSheet } from 'react-native'
 import Icon from 'react-native-vector-icons/MaterialCommunityIcons'
 
+import Typography from '../../components/Typography'
+import { OS } from '../../lib/consts'
 import Colors from '../../themes/colors'
-import { py } from '../../themes/utils'
+import Fonts from '../../themes/fonts'
+import Units from '../../themes/units'
+import { px } from '../../themes/utils'
 
 const styles = StyleSheet.create( {
   chevron: {
     color: Colors.PrimaryText,
   },
   container: {
-    ...py( 21 ),
-    paddingLeft: 12,
+    ...px( Units.HorizontalLayoutMargin ),
+    minHeight: Units.MinimumTouchDimension * Units.LineHeightMultiplier,
     flexDirection: 'row',
     justifyContent: 'space-between',
     alignItems: 'center',
-    borderBottomWidth: 0.5,
+    borderBottomWidth: Units.Separator,
     borderBottomColor: Colors.Separator,
   },
   title: {
     color: Colors.PrimaryText,
-    fontSize: 14,
+    fontSize: Units.Base * Units.GurmukhiLatinRatio,
+    ...( OS.android && { fontFamily: Fonts.MuktaMahee } ),
```

Comment:
Should this be here? Are we going to have to repeat this everywhere else there is Gurmukhi used for android?

Reply:
We decided on Slack to table this conversation for a later time, filing a ticket to spike it together https://github.com/shabados/mobile/issues/224